src/common/dsdl/ocvsmd/common/svc/node/ExecCmd.0.2.dsdl

```diff
@@ -6,6 +6,7 @@ UavcanNodeExecCmdReq.0.1 payload
 
 ---
 
+int32 error_code
```

Comment:
Normally, such change would be a breaking change, BUT we are at `0` major version, so it's fine to change it as needed.